### ISSUE TITLE
Hide Intro Text not working and Read More button is insinde the introtext class

### ISSUE THREAD

**Issue by ChrisWST** [JoomShaper/helix-ultimate#322]
Hello,

I noticed that when I choose in the Joomla Category Blog menu to hide the intro text, it still shows.
In order to make it work, I had to add this code
	**<?php if ($this->item->params->get('show_intro')) : ?>
		<?php echo $this->item->introtext; ?>
	<?php endif; ?>**
Can you please add it as default in your future update?

Also the Read More is inside the introtext div: https://prnt.sc/SRgDA-NdJ8iQ
So if you do not have an introtext on the article, the Read More is not showing.
I hope you can fix these 2 issues in the future update.

**Comment by Savama51** [comment]
Hi @ChrisWST,

Well it seems that's the way it works on basic joomla Cassopia template too !
The setting "hide/show intro text" in blog view acts only in article details view.
I use to add `<hr id="system-readmore" />` at the start of my articles which I don't want to see introtext in category view.

Read more in introtext is a little weird, I agree...

**Comment by ChrisWST** [comment]
You are right, I also used css to hide or show introtext. But on some templates I develop I have different blog styles, ones with intro text, others without, and it would be great to have that option working.
Anyway, as you can see from the picture, it's a simple IF php code you need to add in order to get that going.

Regarding Read More, I think it's a little mistake in Helix Ultimate 2, because in the first version they are separated.
